features/cron.feature

```diff
@@ -2,7 +2,8 @@ Feature: Manage WP-Cron events and schedules
 
   Background:
     Given a WP install
-    And I run `wp config set DISABLE_WP_CRON false --raw --type=constant --anchor="if ( ! defined( 'DISABLE_WP_CRON' ) )"`
+    And I run `wp config set DISABLE_WP_CRON false --raw --type=constant --anchor='// ** MySQL settings - You can get this info from your web host ** //'`
```

Comment:
@johnbillion `--anchor='// ** MySQL settings - You can get this info from your web host ** //'` here is a cause

Reply:
and here is a fix
```suggestion
    And I run `wp config set DISABLE_WP_CRON false --raw --type=constant --anchor='// ** Database settings - You can get this info from your web host ** //'`
```